tutorials/speaker_tasks/Speaker_Identification_Verification.ipynb

```diff
@@ -1,1267 +1,1228 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "speaker_model = nemo_asr.models.EncDecSpeakerLabelModel(cfg=finetune_config.model, trainer=trainer_finetune)\n",
+    "speaker_model.maybe_init_from_pretrained_checkpoint(finetune_config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the config, we keep weights of preprocessor and encoder, and attach a new decoder as mentioned in above section to match num of classes of new data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "uFIOsuFYVLzr"
+   },
+   "outputs": [],
+   "source": [
+    "## Fine-tuning for 5 epochs¶\n",
+    "trainer_finetune.fit(speaker_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Tip: Add more data augmentation and dropout while finetuning on your data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "5DNidtl4VplU"
+   },
+   "source": [
+    "# Saving .nemo file\n",
+    "Now we can save the whole config and model parameters in a single .nemo and we can anytime restore from it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "am5wej6-VdZW"
+   },
+   "outputs": [],
+   "source": [
+    "restored_model.save_to(os.path.join(log_dir_finetune, '..',\"titanet-large-finetune.nemo\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "WnBhFJefV-Pf"
+   },
+   "outputs": [],
+   "source": [
+    "!ls {log_dir_finetune}/.."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "kVx1hNP_V_iz"
+   },
+   "outputs": [],
+   "source": [
+    "# restore from a save model\n",
+    "restored_model = nemo_asr.models.EncDecSpeakerLabelModel.restore_from(os.path.join(log_dir_finetune, '..', \"titanet-large-finetune.nemo\"))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "80tLWTN40uaB"
+   },
+   "source": [
+    "# Speaker Verification"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "VciRUIRz0y6P"
+   },
+   "source": [
+    "Training for a speaker verification model is almost the same as the speaker recognition model with a change in the loss function. Angular Loss is a better function to train for a speaker verification model as the model is trained in an end to end manner with loss optimizing for embeddings cluster to be far from each other for different speaker by maximizing the angle between these clusters"
```

Comment:
The words should be hyphenated if multiple words are used as an adjective:
end to end -> end-to-end